.gitignore

```diff
@@ -24,6 +24,10 @@ test-output/
 *.o
 *.so
 
+# TypeScript built source #
+###########################
+dashboard/src/**/*.js
```

Comment:
I don't have on my side the typescript built source in the same folder so I'm not sure about this exclude ?